packaging/run_nnunet_inference.py

```diff
@@ -119,6 +126,7 @@ def main():
     if args.path_dataset is not None and args.path_images is not None:
         raise ValueError('You can only specify either --path-dataset or --path-images (not both). See --help for more info.')
     
+    inference_mode = None
```

Comment:
This does not seem to be used anywhere. Can be removed I guess

Reply:
Yes good catch.